api/src/shared/application/scripts/script-runner.js

```diff
@@ -0,0 +1,60 @@
+import * as url from 'node:url';
+
+import pick from 'lodash/pick.js';
+import { hideBin } from 'yargs/helpers';
+import yargs from 'yargs/yargs';
+
+import { disconnect } from '../../../../db/knex-database-connection.js';
+import { learningContentCache } from '../../infrastructure/caches/learning-content-cache.js';
+import { logger as defaultLogger } from '../../infrastructure/utils/logger.js';
+
+function isRunningFromCLI(scriptFileUrl) {
```

Comment:
Je préconise d’utiliser toujours le camelCase pour les acronymes et les sigles. D'ailleurs on a `scriptFileUr`l et non `scriptFileUrl` :
```suggestion
function isRunningFromCli(scriptFileUrl) {
```

En effet utiliser du camelCase pour les acronymes et les sigles permet d'avoir une règle de nommage systématiquement cohérente et automatisable.